tools/runners/Slang.py

```diff
@@ -39,10 +41,9 @@ def prepare_run_cb(self, tmp_dir, params):
         for define in params['defines']:
             self.cmd.extend(['-D', define])
 
-        # hdlconv and utd tests are not semantically valid SystemVerilog, so we
-        # can only expect to run parsing successfully.
         tags = params["tags"]
-        if "hdlconv" in tags or "hdlconv_std2012" in tags or "hdlconv_std2017" in tags or "utd-sv" in tags:
+
+        if params['mode'] in ["parsing", "preprocessing"]:
```

Comment:
--parse-only is mutually exclusive with -E (which is just a shortcut for --preprocess-only) which gets set above. I think some of the preprocessor tests aren't even valid syntactically and so can't be parsed.

Reply:
Right, thanks!